code/script.js

```diff
@@ -1,51 +1,246 @@
+      <option id="littleMermaid" value="littleMermaid">The Little Mermaid (2023)</option>
+      <option id="alfons" value="alfons">Aja baja Alfons Åberg</option>
+      <option id="meg" value="meg">Meg 2: The Trench</option>
+    </select>
+  `;
+   // Drop-down menu with different choices.
+   const movieOptions = document.getElementById('movieOptions');
+   movieOptions.addEventListener('change', (event) => {
+     movieOption = event.target.selectedOptions[0].text;
+     showMessage(`${movieOption}`, "user");
+     setTimeout(() => showtime(), 1000);
+   });
+};
+
+const showtime = () => {
+  removeChildrenFunction();
+  showMessage(`Good choice! Select one of the available showtimes for ${movieOption}.`, 'bot');
+  // Three new buttons
+  inputWrapper.innerHTML += `
+    <button id="showtimeOne" type="submit">12:00 PM</button>
+    <button id="showtimeTwo" type="submit">3:00 PM</button>
+    <button id="showtimeThree" type="submit">6:00 PM</button>`;
+  const showtimeOne = document.getElementById('showtimeOne');
+  const showtimeTwo = document.getElementById('showtimeTwo');
+  const showtimeThree = document.getElementById('showtimeThree');
+  //Eventlisteners for buttons
+  showtimeOne.addEventListener('click', () => {
+  showtimeOption = reservation.showtimesAvailable[0];
+  handleShowtime();
+  });
+  showtimeTwo.addEventListener('click', () => {
+  showtimeOption = reservation.showtimesAvailable[1];
+  handleShowtime();
+  });
+  showtimeThree.addEventListener('click', () => {
+  showtimeOption = reservation.showtimesAvailable[2];
+  handleShowtime();
+  });
+  // Function that shows a message with the user's order and also calls the function for the next question.
+  const handleShowtime = () => {
+  showMessage(`${showtimeOption}`, "user");
+  setTimeout(() => confirmReservation(), 1000);
+  };
+};
 
-// Set up your eventlisteners here
+const confirmReservation = () => {
+  removeChildrenFunction();
+  showMessage(`Nice ${customerName}! You want to make a reservation for ${movieOption} at ${showtimeOption} for a ${ticketOption.toLowerCase()}. It will cost US$ ${price}.`, "bot");
+  setTimeout(() => {
+    showMessage(`Would you like to confirm your order?`, "bot");
+  }, 1000);
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
-setTimeout(greetUser, 1000)
+  inputWrapper.innerHTML += `
+    <button id="yes" type="submit">Yes</button>
+    <button id="no" type="submit">No</button>`;
+  document.getElementById('yes').addEventListener('click', () => {
+    showMessage(`Your reservation is complete. Enjoy the movie!`, "bot");
+    reloadPage();
```

Comment:
Using the reloadPage() is a great way of finishing of the interaction with the bot. I liked that the user has to start the chat by pressing the tickets-button and that the session ends with the bot disappearing. Gives the user a really good overall impression.